### ISSUE TITLE
prop_dynamics randomly don't animate!

### ISSUE THREAD

**Issue by 111112oo** [tf2classic/tf2classic-issue-tracker#347]
On some maps I run and want to run on my server, there are animated prop_dynamics that randomly don't animate. They look like they play the first frame or so of their animation and then freeze up when this happens, and then work fine other times. This issue does not occur at all in live TF2.

Here is a test map where this occurs, copying a setup from a map I worked on and want to run:
[test_tf2canimations.txt](https://github.com/tf2classic/tf2classic-issue-tracker/files/15176324/test_tf2canimations.txt)
Rename to .vmf (plaintext format), you will also need live TF2 models mounted.

What's weird is that this bug only occurs under certain conditions, and I have no idea what they are! The hammer models in the white room you spawn in of the test map will bug and randomly not animate. But in the grey rooms in the map, the bug does not occur with the exact same hammer models!

I run a deathrun server and models animating properly is very important there, as deadly obstacles are commonly tied to animated models. If the animation doesn't play then the hurt triggers will not line up (and maybe not move with the animation if attachment parented, haven't tested this) which will lead to much confusion.

**Comment by newgreenshoot** [comment]
Our guess is that this is related to dynamic props not transmitting to players outside of their PVS. If so, we could add a Hammer keyvalue to force them to always transmit to clients.

Presuming that's the issue though, it should affect live TF2 as well, so it's strange that the behavior's inconsistent.

**Comment by 111112oo** [comment]
> Our guess is that this is related to dynamic props not transmitting to players outside of their PVS. If so, we could add a Hammer keyvalue to force them to always transmit to clients.
> 
> Presuming that's the issue though, it should affect live TF2 as well, so it's strange that the behavior's inconsistent.

I do not think this is the case at all. The test map I provided has this issue in an empty cube room with no obstructions. You can stand perfectly still and watch the props animate then randomly not animate. Wouldn't the props not be loaded at all if you weren't in the PVS? And usually if you enter a prop's visible space mid animation, the animation still plays, just desynced, unless you have the "realtime" parameter in the prop's qc which does what you suggested but is of course expensive. Also also, the prop does play the first frame or so of the animation before failing. I can't right now but try compiling the test map without vvis and see if it still happens.
